### DISCUSSION TITLE
Cloudflare Pages example

### DISCUSSION THREAD

**Discussion by wadclapp** [decaporg/decap-cms#7155] [Q&A]
[Cloudflare Pages](https://developers.cloudflare.com/pages/) is very popular for static sites.

But there doesn't seem to be any documentation on how to use it with Decap CMS?

I just need a simple CMS solution that a user (non-technical content editor) can log in with username/password, and any changes get saved to Github repo, which triggers the website on Cloudflare Pages to be re-generated (SSG) with the changes and re-deployed.

Is that possible? Is there any guide available on how to simply set this up?

Can it be done with [Cloudflare Access](https://developers.cloudflare.com/pages/functions/plugins/cloudflare-access/) or [Cloudflare Zero Trust](https://developers.cloudflare.com/cloudflare-one/)??

Thank you in advance for any help.

**Comment by wadclapp** [comment]
Opted for solution here https://github.com/i40west/netlify-cms-cloudflare-pages for now which allows "Login with Github", which is the only solution/guide I found for Cloudflare Pages, but it's not ideal as it means if I want to give a Content Editor access they have access to the project's Github source code etc?

**Reply by TravisBrace** [reply]
Currently how we're doing it too but I think you can limit a users role and enable the editorial mode which will create a new branch for each page.

Now what I can't figure out is how to connect the cloudflare pages link generated to the show preview function in decapCMS.

**Reply by wadclapp** [reply]
Thanks for the tip, I was considering doing something like that.

I've disabled preview links in the config for my project, and I haven't attempted to set it up, so I can't help there unfortunately!
